tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8,py27,py34
+envlist = flake8,py34
```

Comment:
I would prefer to have both environments - py27 and py34 - running tests. No reason to focus only on py34 just now